app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ActiveRecord::Base
+  has_secure_password
+  has_many :bucketlists
+
+  validates_presence_of :firstname, :lastname, :email
+  validates_uniqueness_of :email
+  validates_presence_of :password, on: :create
+
+  before_save :generate_iss_number
+
+  private
+
+  def generate_iss_number
```

Comment:
JWT has the iss(issuer) claim which can be used to verify the token. 
See: https://tools.ietf.org/html/rfc7519#section-4.1.1
In this app it is being used to invalidate all active tokens when it is reset using the logout action.
Thanks.